code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm

```diff
@@ -4,7 +4,7 @@
 	upgrade_name = ""
 	caste_desc = "The mind of the hive"
 	caste_type_path = /mob/living/carbon/xenomorph/hivemind
-	tier = XENO_TIER_THREE
+	tier = XENO_TIER_ZERO
```

Comment:
Any reason why it is zero and not four..?

Reply:
Hivemind still has a dedicated line in the hive status for itself, so I just updated the castedatum to match the mob.

Plus imo it's not really T4 caste material being basically just a disembodied drone with big voice.

Reply:
As far as I see, tier 0 doesn't count towards the total hive count, so it'd possibly remove some T2/T3 slots.

Reply:
Don't worry it counts, just like larva.